ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -51,6 +51,8 @@ class ClientQueryViewController: UITableViewController, Themeable {
 	private var observerContext : UnsafeMutableRawPointer
 	var refreshController: UIRefreshControl?
 
+	var interactionController: UIDocumentInteractionController?
+
 	// MARK: - Init & Deinit
 	public init(core inCore: OCCore, query inQuery: OCQuery) {
 		observerContext = UnsafeMutableRawPointer(&observerContextValue)
```

Comment:
@pablocarmu I checked the code and I think `observerContext` is not really required, since it is used when e.g. you observe two objects for the change of same keypath and want to differentiate between them when the observer iis called.